src/sources/webgl.ts

```diff
@@ -24,6 +24,7 @@ type WebGlExtensionsPayload = {
   shaderPrecisions: string[] // ['FRAGMENT_SHADER.LOW_FLOAT=127,127,23...
   extensions: string[] | null // ['ANGLE_instanced_arrays', 'EXT_blend_minmax', 'EXT_color...
   extensionParameters: string[] // ['COMPRESSED_RGB_S3TC_DXT1_EXT(33776)', 'COMPR...
+  unsupportedExtensions: string[]
```

Comment:
Please add a note like for the other fields.

In general, try to keep the code you add at least as good as the code around.